tests/unit/lib/cookiecutter/test_question.py

```diff
@@ -0,0 +1,74 @@
+from unittest import TestCase
+from samcli.lib.cookiecutter.question import Question, QuestionKind
+
+
+class TestQuestion(TestCase):
+
+    _ANY_TEXT = "any text"
+    _ANY_KEY = "any key"
+    _ANY_OPTIONS = ["option1", "option2", "option3"]
+    _ANY_ANSWER = "any answer"
+    _ANY_NEXT_QUESTION_MAP = {
+        "option1": "key1",
+        "option2": "key2",
+        "option3": "key3",
+    }
+    _ANY_DEFAULT_NEXT_QUESTION_KEY = "default"
+    _ANY_KIND = QuestionKind.default
+
+    def setUp(self):
+        self.question = Question(
+            text=self._ANY_TEXT,
+            key=self._ANY_KEY,
+            options=self._ANY_OPTIONS,
+            default=self._ANY_ANSWER,
+            is_required=True,
+            next_question_map=self._ANY_NEXT_QUESTION_MAP,
+            default_next_question_key=self._ANY_DEFAULT_NEXT_QUESTION_KEY,
+            kind=self._ANY_KIND,
+        )
+
+    def test_creating_questions(self):
+        q = Question(text=self._ANY_TEXT, key=self._ANY_KEY)
+        assert q.text == self._ANY_TEXT
+        assert q.key == self._ANY_KEY
+        assert q.options == []
+        assert q.default_answer == ""
+        assert q.required is False
+        assert q.next_question_map == {}
+        assert q.default_next_question_key is None
+        assert q.kind == QuestionKind.default
```

Comment:
We should use `self.assertEqual()` and other `self.assert*()` instead of `assert`. The former will output more helpful information in case of a failure. https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual